ci/install-build-deps.sh

```diff
@@ -1,62 +1,107 @@
 #!/bin/bash
 
 # This script is meant to be sourced.
+#
```

Comment:
Instead of comments in the file, can we show the warning to users and prompt for continue?

Reply:
A script cannot know whether it's being sourced or not.